web/models/students.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CoursePlanner.Students do
+  @moduledoc """
+    Handle Students specific logics
+  """
+  alias CoursePlanner.Repo
+  alias CoursePlanner.User
+  import Ecto.Query
+  alias Ecto.{Changeset, DateTime}
+
+  @students from u in User, where: u.role == "Student" and is_nil(u.deleted_at)
+
+  def all do
+    Repo.all(@students)
+  end
+
+  def update(id, params) do
+    case Repo.get(User, id) do
+      student ->
+        student
+        |> User.changeset(params, :update)
+        |> add_timestamps()
```

Comment:
Doing that would require me to refactor the part of the code that I'm not touching, hence do it later when other things are merged.